src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -445,6 +448,17 @@ function ComposerWithSuggestions({
         textInputRef.current.blur();
     }, []);
 
+    useEffect(() => {
+        // Initial focus ref to prevent unneccessary focus after first render.
+        if (initialFocusedRef.current || isNativeApp || !shouldAutoFocus) {
```

Comment:
I don't think `initialFocusedRef` is needed anymore since we removed focus calls.

Reply:
I don't think the `isNativeApp` check is really necessary here either. Also not sure if you considered under what cases `shouldAutoFocus` would be false and if we'd not want to scroll to the bottom? It seems to me like we **always** want to be scrolled to the bottom, but I could be missing some obvious case!